packages/feedback/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "feedback",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~8.2.14",
+    "@angular/common": "~8.2.14",
+    "@angular/compiler": "~8.2.14",
+    "@angular/core": "~8.2.14",
+    "@angular/forms": "~8.2.14",
+    "@angular/platform-browser": "~8.2.14",
+    "@angular/platform-browser-dynamic": "~8.2.14",
+    "@angular/router": "~8.2.14",
+    "angular-truncate-pipes": "0.0.7",
+    "apollo-angular": "^1.8.0",
+    "apollo-angular-link-http": "^1.9.0",
+    "apollo-cache-inmemory": "^1.6.0",
+    "apollo-client": "^2.6.0",
```

Comment:
Apollo and its dependencies have upgraded can you please upgrade this all

https://www.npmjs.com/package/apollo-client

Also please check all the dependencies which is outdated.

Reply:
you mean *npm update*  thats how i can update them ?

Reply:
As you suggested uninstall then install.